pallets/omnipool/src/lib.rs

```diff
@@ -468,10 +468,10 @@ pub mod pallet {
 
 			let amount = T::Currency::free_balance(asset, &Self::protocol_account());
 
-			ensure!(
-				amount >= T::MinimumPoolLiquidity::get() && amount > 0,
-				Error::<T>::MissingBalance
-			);
+			let ed = T::AssetRegistry::existential_deposit(asset).unwrap_or(u128::MAX);
```

Comment:
maybe it would be better to return an error here.

because the error returned would be potentially misleading ( it would return MissingBalance).